packages/web/src/app/App.tsx

```diff
@@ -46,6 +58,11 @@ export const AppInner = () => {
           merchantId={MERCHANT_ID || ''}
           coinflowEnv={IS_PRODUCTION ? 'prod' : 'sandbox'}
         />
+        <AppModal
```

Comment:
Why here and not with the rest of the common modals?

Reply:
because the page is outside the "WebPlayer", meaning no left nav, no bottom playbar. confirmed with @dylanjeffers on this too by the way